pkg/solana/chainreader/chain_reader_test.go

```diff
@@ -991,6 +1041,11 @@ func (s *skipEventsChainReader) GetLatestValue(ctx context.Context, readIdentifi
 		s.t.Skip("Events are not yet supported in Solana")
 	}
 
+	// Isolate test until we have InputModifications
+	if strings.Contains(s.t.Name(), "wraps_config_with_modifiers_using_its_own_mapstructure_overrides") {
+		s.t.Skip("Isolating test until we have InputModifications")
+	}
+
```

Comment:
The test is still being skipped, just delete this Skip and everything should work
```suggestion
```